### PR TITLE
fix: fix bug cellEditorParams not set when field is child of column g…

### PR DESCRIPTION
**Pull request by Guismus**
…roup

(For testing, use cypress test column grouping and console.log formated lines)

### CODE REVIEW

File: src/inputs/Table/Table.js [hunk 1]
```diff
@@ -102,8 +102,12 @@ const _moveExtraPropertiesToCellEditorParams = (col) => {
 
 const _formatColumnsData = (columns, dateFormat) =>
   clone(columns).map((col) => {
-    _formatMinMaxDatesInColumns(col, dateFormat);
-    _moveExtraPropertiesToCellEditorParams(col);
+    if (col.children) {
+      col = { ...col, children: _formatColumnsData(col.children, dateFormat) };
```
Comment (csm-thu) [inline src/inputs/Table/Table.js:106]: Could be simplified by using instead:
`col.children = _formatColumnsData(col.children, dateFormat);`

File: src/inputs/Table/Table.js [hunk 2]
```diff
@@ -102,8 +102,12 @@ const _moveExtraPropertiesToCellEditorParams = (col) => {
 
 const _formatColumnsData = (columns, dateFormat) =>
   clone(columns).map((col) => {
-    _formatMinMaxDatesInColumns(col, dateFormat);
-    _moveExtraPropertiesToCellEditorParams(col);
+    if (col.children) {
```
Comment (nborde-CSM) [inline src/inputs/Table/Table.js:105]: Maybe use a ternary form instead of _if else_ ?

**Review by nborde-CSM — APPROVED** [review]
One remark but LGTM :+1:

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Review by esasova — APPROVED** [review]
LGTM :+1:

**Pull request merged**